app/routes/dashboard.php

```diff
@@ -66,8 +66,12 @@
     Route::group(['prefix' => 'notifications'], function () {
         Route::get('/', [
             'as'   => 'dashboard.notifications',
-            'uses' => 'DashboardController@showNotifications',
+            'uses' => 'NotificationController@showSlack',
         ]);
+        Route::get('slack', 'NotificationController@showSlack');
```

Comment:
L67 defines `/notifications` to be routed to `NotificationContoller@showSlack`
L71 defines `/notifications/slack` to be routed to the same function `NotificationController@showSlack` 

I'm not happy with that, I'd prefer to have only one route(`/notifications/slack`) to go to `NotificationController@showSlack` but if the user comes from /notifications I want to redirect it to /notifications/slack. A kind of alias I presume, but I don't know how to do that with Laravel's router.